packages/hint-axe/src/hint.ts

```diff
@@ -11,9 +11,9 @@
 
 import { AxeResults, Result as AxeResult, NodeResult as AxeNodeResult } from 'axe-core';
 
-import { debug as d } from 'hint/dist/src/lib/utils/debug';
+import { debug as d } from '@hint/utils/dist/src/debug';
+import { readFileAsync } from '@hint/utils/dist/src/fs/read-file-async';
```

Comment:
Should this be `from '@hint/fs|debug'` or something like that? I think is the first time I see you link to `@hint/utils/dist`

Reply:
If the rule is used in the browser extension, we need to link directly to the file we want to use :(